README.md

```diff
@@ -27,60 +74,106 @@ mutation SignInMutation($input: signInInput!) {
+**Example:**
 
-Once the salt is retrieved the hash password can be computed with bcrypt. In JavaScript:
-
-```javascript
-import bcrypt from 'bcryptjs';
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'x-csrf-token: <YourCSRFTokenValue>' \
+-H 'cookie: _sorare_session_id=<YourCookieValue>' \
+-d '{
+  "operationName": "SignInMutation",
+  "variables": { "input": { "email": "<YourEmail>", "password": "<YourHashPassword>" } },
+  "query": "mutation SignInMutation($input: signInInput!) { signIn(input: $input) { currentUser { slug email } errors { message } } }"
+}'
 
-bcrypt.hashSync(password, salt);
+{"data":{"signIn":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>"},"errors":[]}}}
 ```
 
-You'll also need to pass a `_sorare_session_id` cookie and a `X-CSRF-TOKEN` token header. They can be retrieved with the same request as the salt.
+Should you keep the same `cookie` for future API calls, they will then be authenticated as this user.
 
-### With a cookie
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'x-csrf-token: <TheSameCSRFTokenValue>' \
+-H 'cookie: _sorare_session_id=<TheSameCookieValue>' \
+-d '{
+  "operationName": "CurrentUserQuery",
+  "query": "query CurrentUserQuery { currentUser { slug email } }"
+}'
 
-To use a cookie authentication you need to store the new `_sorare_session_id` and `X-CSRF-TOKEN` and pass it to all next requests.
+{"data":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>"}}}
+```
+
+This is exactly the authentication mechanism that happens while browsing the [sorare.com](https://sorare.com) website.
+
+### GraphQL `signIn` mutation & JWT tokens
 
-### With a JWT token
+A better way than cookies to stay authenticated is to rely on [JWT tokens](https://jwt.io/).
 
-To authenticate by JWT you need to request a valid JWT token in the `signIn` mutation :
+We provide JWT tokens within the `signIn` mutation. They can be retrieve using the following mutation:
 
 ```gql
-  mutation SignInMutation($input: signInInput!) {
-    signIn(input: $input) {
-      currentUser {
-        slug
-        jwtToken(aud: "YOUR_AUD") {
-          token
-          expiredAt
-        }
+mutation SignInMutation($input: signInInput!) {
+  signIn(input: $input) {
+    currentUser {
+      slug
+      jwtToken(aud: "<YourAud>") {
+        token
+        expiredAt
       }
-      ...
     }
   }
+}
 ```
 
-`YOUR_AUD` is a mandatory string parameter that helps identifying your app. Also, tokens generated with an `aud` are scoped to this `aud`. We recommend to use an `aud` reflecting the name of your app - like `myrealappname` - to make it easier to debug & track.
+`<YourAud>` is a mandatory _string_ parameter that helps (us & you) identifying your app. We recommend to use an `aud` reflecting the name of your app - like `myappname` - to make it easier to debug & track.
```

Comment:
_`<YourAud>` is a required _string_ parameter that helps us, and you, identify your app. We recommend using an `aud` related to your app's name to make it easier to debug and track  (e.g.: `myappname`)._